client/app/lib/redux/dispatchInitialActions.coffee

```diff
@@ -59,7 +59,7 @@ module.exports = dispatchInitialActions = (store) ->
 
   promise = loadAccount(store)
     .then -> loadGroup(store)
-    .then -> loadUserDetails(store)
+  # .then -> loadUserDetails(store)
```

Comment:
@usirin you were going to delete this requirement for payment flow afair, can u chime in?